active/diversity-inclusion.md

```diff
@@ -0,0 +1,68 @@
+- Performing Audit or report on D&I of Django events and contributors. 
+- Supporting the organisation of regional Django related events around the world in line with the group's purpose.
+- Providing support to the DSF board of directors in managing the DSF Membership and the DSF members.
+- 
+
+## Initial membership
+
+- Chair: Ariane Djeupang
+- Co-Chair: Keanya Phelps
+- Board Liaison: Afi Maame
+- Other members:
+   - Eli Rosselli
+   - Priya Pahwa
+   - TBD 
+
+## Future membership
+
+- Folks that would like to join the Diversity and Inclusion Work Group should send an email to the WG `diversity-inclusion@djangoproject.com` introducing themselves. The WG will vote on new member requests verbally at meetings, and the board can take part to the vote.
+- The working group adopts the DSF [Code of Conduct](https://www.djangoproject.com/conduct/) .
+- Any actions made against the principles in the Code of Conduct (found by a simple majority of 51%+) will result in the acting person being removed from the working group.
+- Diversity and Inclusion working group voting members are required to confirm their intent to continue participating in the Working Group within one month of a new DSF Board election. This will be led by chair/vice-chair. If there is no response within two weeks, then the individual is automatically removed.
+- A member of the Diversity and Inclusion Working Group, can step down at any time, and the working group will work to replace that voting member.
+- The Chair / Co-Chair will be responsible for coordinating the group, scheduling meetings, renewing the group’s membership, and ensuring that the group’s activities align with its scope and responsibilities.
+- 
+
+## Budget
+
+- TBA
+
+## Comms
+
+- The team will communicate via the `diversity-inclusion-wg@djangoproject.com` mailing list, through Discord and Slack.
+- If needed, conference calls may be set up via Google Meet, Microsoft Teams or Zoom.
+- The method of voting on projects and proposals will be determined by the working group.
+- Preferred method of voting will be electronically (email or chat) and the vote results will be reported via email to the board in the monthly report.
+- Asynchronous communication will be preferred as membership will be in many different time zones.
+- 
+
+
+## Reporting
+
+- The Diversity and Inclusion Working Group will submit by email a monthly report to the DSF board of Directors.
+
+
+## Appendix
+
+### Decision Making Procedures
+- Discussions will happen via online communication platforms, video chat, or mailing list.
+- With respect to new policy recommendations:
+  - The working group may solicit feedback on proposed drafts.
+  - The working group will consider outside input, edit the document, then vote for final approval.
+  - The working group will vote on new standards and will require a quorum of two-thirds of the voting members for approval of motions / content.
```

Comment:
I don’t understand the difference between "final proposal" and "new standards" when it comes to "new policy recommendations"?